air-script/tests/aux_trace/aux_trace.air

```diff
@@ -14,15 +14,15 @@ boundary_constraints:
     enf p0.first = 1
     enf p0.last = 1
 
-    # auxiliary boundary constraint with a random value
-    enf p1.first = $rand[0]
+    # auxiliary boundary constraint
+    enf p1.first = 1
```

Comment:
I said the wrong thing in my previous review, sorry. I had misunderstood that we're removing the default `$rand` option and making the declaration required.

Given that, I'd like to keep the random values in this test, so instead of removing these, we should add the `random_values` declaration (as you did the first time). Sorry for the mistake!